kbfsgit/runner.go

```diff
@@ -450,6 +453,17 @@ func (r *runner) processGogitStatus(
 	lastByteCount := 0
 	for update := range statusChan {
 		if update.Stage != currStage {
+			if r.verbosity >= 3 {
+				f, err := os.Create(filepath.Join(
+					os.TempDir(), fmt.Sprintf("memprof.%d.prof", update.Stage)))
+				if err != nil {
+					r.log.CDebugf(ctx, err.Error())
+				} else {
```

Comment:
might want to try `runtime.GC()` here, otherwise profiles could be skewed by garbage.

Reply:
Thanks, done.